connectors/sources/gcs.py

```diff
@@ -44,9 +44,7 @@
 DEFAULT_WAIT_MULTIPLIER = 2
 DEFAULT_CONTENT_EXTRACTION = True
 DEFAULT_FILE_SIZE_LIMIT = 10485760
-STORAGE_EMULATOR_HOST = os.getenv(
-    key="STORAGE_EMULATOR_HOST", default=None
-)  # For end to end tests set this environment variable with target host.
+STORAGE_EMULATOR_HOST = os.environ.get("STORAGE_EMULATOR_HOST")
```

Comment:
I assume this will return `None` if the env variable is not found... but just want to make sure (as my python is a bit rusty ;) )

Reply:
yes this is correct.

```
>>> import os
>>> print(os.environ.get("I DONT EXIST"))
None
```